src/script.ts

```diff
@@ -90,22 +90,18 @@ var L10N = {
   },
 }
 
-var $root = document.documentElement
-var $$hidden = []
+let $root = document.documentElement
```

Comment:
do we want to use `const` instead of `let` if we are not modifying the variable value later? (applies to the rest of the variables in the file as well)

Reply:
Yes, fair enough.